tests/testthat/test-dplyr.R

```diff
@@ -122,13 +122,20 @@ test_that("'sample_n' and 'sample_frac' work in nontrivial queries (#1299)", {
 })
 
 test_that("'sdf_broadcast' forces broadcast hash join", {
-  skip_databricks_connect()
+  if (sc$method == "databricks-connect") {
```

Comment:
For consistency isn't it better to use `is_testing_databricks_connect()` instead of `sc$method == "databricks-connect"`?